source/documentation/API-reference.html.md.erb

```diff
@@ -46,6 +46,66 @@ This explains the steps shown in the above diagram on the access flow for Trader
+
+### Dynamic endpoints
+
+The TGP sandbox contains a test data store allowing you to create and manipulate data as part of the user journey. The following endpoints support dynamic behaviour:
+
+- Create a TGP goods record
+- Update a TGP goods record
+- Get TGP goods records
+- Get a single TGP goods record
+- Remove a TGP goods record
+- Maintain TGP profile
+
+Records created and updated within the data store will persist for one year.
+
+Schema validation is applied to the endpoints in the same way as the production endpoints, so attempting to create or update a record with invalid data will result in an error being returned.
+
+### Creating a new TGP for test users in the dynamic sandbox
+
+In the <u>sandbox environment only</u>, the Maintain TGP Profile endpoint can be used to create new TGPs for test users you’ve set up in the [Developer Hub][DEV_HUB].
+
+To create a new TGP, you’ll need to use the test user’s EORI number as the eori in the request as well as the actorId. You’ll also need to provide a number matching the required format for a UKIMS number to create the profile. Optionally you can also provide dummy data in the correct format for NIRMS or NIPHL numbers.
+
+Once created, you’ll be able to create and manipulate data for this test user’s EORI within the dynamic sandbox.
+
+*Note: the Maintain TGP Profile can not be used to create new profiles in the production environment.*
+
+### Static endpoints
+
+The following endpoints do not provide dynamic behaviour:
+
+- Request advice
+- Withdraw advice
+
+When testing the functionality for these endpoints you should refer to [Trader Goods Profile stubs][TGP_STUB] for details of test data.
+
+### Admin API
+
+*Note: The TGP Admin API exists purely within the sandbox environment and does not exist in the production environment.*
+
+The Admin API allows you to update data values which are not typically possible from the client side. This is because they are applied by the HMRC systems in response to specific triggers or business rules.
+
+- Commodity code has expired in Trade Tariff
+    - a goods record should have the toReview value updated to true with a reviewReason of “The commodity code has expired. You’ll need to change the commodity code and categorise the goods.”
+
+- Measures have changed in Trade Tariff
+    - the goods record should have the toReview value updated to true with a reviewReason of “The restrictions have changed or there may be new restrictions. You need to categorise the record.”
+
+- Record is locked due to an open advice request
+    - the goods record should have the adviceStatus value updated to “Requested”, “In progress” or “Information requested” and the locked value should be true.
+
+- Record has a different declarable state
+    - the goods record should have the declarable value updated to “Ready for IMMI”, “Not ready for IMMI” or “Not ready for use”.
+
+Refer to the Admin API endpoint specification for more information.
```

Comment:
guessing its the maintain profile endpoint that we call it admin api unless we have specific admin api ?

Reply:
There's a specific admin api endpoint that allows you to adjust values that would normally be adjusted by business logic

Reply:
https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/trader-goods-profiles-test-support/1.0